updating Windows Server Core limitations and FOD (#5344)
MicrosoftDocs/windowsserverdocs: WindowsServerDocs/administration/server-core/what-is-server-core.md

| Internet Explorer                  | with FOD 1903   | available                      |
| Edge                               | not available   | available**                    |
| Edge Chromium (Enterprise)         | optional install| optional install               |
| Built-in help system               | not available   | available                      |
| Windows 10 Shell                   | not available   | available                      |
| Windows Media Player               | not available   | available                      |
| PowerShell                         | available       | available                      |
| PowerShell ISE                     | FOD 1903***     | available***                   |
| PowerShell IME                     | available       | available                      |
| Mstsc.exe                          | not available   | available                      |
| Remote Desktop Services            | available       | available                      |
| Hyper-V Manager                    | FOD 1903        | available                      |
| WordPad\*                          | not available   | available                      |

> \* To read .RTF  files locally stored on a Server Core SKU, users can copy the file(s) to a different Windows computer where WordPad is present.

Reviewer: Cosmetic: redundant space.
```suggestion
\* To read .RTF files locally stored on a Server Core SKU, users can copy the file(s) to a different Windows computer where WordPad is present.
```

> Features indicated with 2 asterisks ( \** ) are end of support by March 21, 2021. Edge Chromium is a more versatile and modern Browser to replace Internet Explorer and EdgeHTML.

Reviewer: Some phrasing and expressions are more common and in more frequent use than others. Here is a suggestion:
```suggestion
Features indicated with 2 asterisks ( \** ) reached end of support by March 21, 2021. Edge Chromium is a more versatile and modern Browser, replacing Internet Explorer and EdgeHTML.
```
By the way, would it be practical to mention Microsoft Edge (Legacy), or is that too soon in this context?

> Features indicated with 3 asterisks ( \*** ) are officially deprecated. Please consider to use Visual Studio Code instead of ISE.

Reviewer: Minor phrasing detail:
```suggestion
Features indicated with 3 asterisks ( \*** ) are officially deprecated. Please consider using Visual Studio Code instead of ISE.
```

> >[!NOTE]

Reviewer: ```suggestion
> [!NOTE]
```

> **Adding Features on Demand can lower security and increase the amount of servicing due security updates needed OS restarts.**

> > The idea of installing of Features on Demand on Server Core is to a balance between security and a smaller footprint and application compatibility.

Reviewer: Natural language/readability:
```suggestion
> The idea of installing Features on Demand on Server Core is to a balance between security, a smaller footprint, and application compatibility.
```

> 

> > We do not encourage to install Features on Demand per default, nor to improve local administration capabilities of Server Core, but to mitigate possible application compatibility issues, of 3rd party applications hosted on a Server Core installation, which might only run as intended when some of the removed featuresets of Server Core is added back by installing Features on Demand.

Reviewer: Syntax/grammar details:
```suggestion
> We do not encourage installing Features on Demand by default, nor to improve local administration capabilities of Server Core, but to mitigate possible application compatibility issues, of 3rd party applications hosted on a Server Core installation, which might only run as intended when some of the removed feature sets of Server Core is re-added by installing Features on Demand.
```

> 

> > For modern Remote Administration please use in-built tools like PowerShell 5.1, Sconfig. or external Tools like web-based Windows Admin Center, or PowerShell 7 for enchanced capabilites.

Reviewer: ```suggestion
> For modern Remote Administration, please use built-in tools like PowerShell 5.1 or Sconfig.exe, or use external Tools, like web-based Windows Admin Center or PowerShell 7, for enhanced capabilities.
```

> 

> > Alternatively you can use Server Manager with RSAT Tools on a current remote management Client or current Windows Server with a graphical experience.

Reviewer: ```suggestion
> Alternatively, you can use Server Manager with RSAT Tools on a current remote management Client or on a current Windows Server with Desktop Experience.
```
